### DISCUSSION TITLE
PROPOSAL: The Third CRO Community Burn on Cronos POS Chain

### DISCUSSION THREAD

**Discussion by lezzokafka** [crypto-org-chain/chain-main#1093] [Ideas]
<img width="1463" alt="Screenshot 2024-11-12 at 09 44 49" src="https://github.com/user-attachments/assets/51445623-01ce-478c-9bba-0b9ae230e6aa">

# Introduction
Following the first and second successful burn of 50 Million $CRO, we are proposing to initiate the third burn of 50 Million of $CRO on the Cronos POS Chain. Once this proposal is passed, an additional 50 Million of $CRO will be burned -  **Leading to a total of 150 Million $CRO permanently removed from the circulating supply.**

## Background
As introduced in the previous proposal, the "Burning Mechanism of $CRO" has been implemented and has been continuously allocating a portion of the $CRO generated through emissions to the community pool at the Cronos POS Chain protocol level. The accumulated amount is then periodically burned to complete the cycle. At the time of writing, the community pool holds approximately [70 Million](https://burninfo.cronos-pos.org/) of $CRO and this proposal suggests a burning of 50 Million from the amount.

# Technical Details
To execute the "burn”, a "community pool spend" governance proposal with a target address needs to be initiated. In particular, funds from the community pool are proposed to be moved to the "dead" address (i.e.[cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv](https://cronos-pos.org/explorer/account/cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv)), which no one has control over, and effectively removes these funds from the circulating supply permanently. The current pending and burned amount, as well as historical burn events, are on the [burn info page](https://burninfo.cronos-pos.org/). Moreover, we can use this [endpoint](https://cronos-pos.org/explorer/api/v1/supply/circulating) to query the current value of the circulating supply at any time, note this number is provided in `basecro`, where `1 CRO = 10^8 basecro` on the Cronos POS chain.

# Governance Votes
After discussion and suggestion to this proposal, an on-chain network upgrade governance proposal will be raised. The following items summarize the voting options and what they mean for this proposal.
- YES: You approve of the proposal on the changes to proposed changes to move 50 Million (`50,000,000`) of $CRO from the community pool to the “dead” address and remove this amount from the circulating supply of CRO.
- NO: You disapprove of the proposal in its current form. Feel free to give feedback and suggestions on how to improve the proposal.
- NO (VETO): You veto the entire motivation for the proposal and are strongly opposed to its implementation.
- ABSTAIN: You are impartial to the outcome of the proposal. If the quorum is not met or less than 1/3 of the participating voting power has voted No, the proposal fails and is considered to be vetoed. In this case, the proposal deposit is burned rather than being returned to respective depositors.

## Let us know your thoughts
The importance of active community participation in shaping the future of the Cronos POS chain cannot be overstated. Engage with your delegated validators and participate in proposal discussions on Discord and GitHub. Your input is vital in fostering a vibrant that benefits all $CRO holders and the wider ecosystem.

**Comment by TexMint** [comment]
I like it. Burn another 50M

**Comment by Cryptokyra** [comment]
Agree with it too

**Comment by Athiboi** [comment]
Burn!We need to adopt same burn mechanism like BNB and make CRO great again!

**Comment by jfcolombel** [comment]
Agree! Burn.

**Comment by abelhd** [comment]
yes burn burn

**Comment by crunchFiveM** [comment]
YES! Burn 100m, 200m idc, but you aint getting my coins!

**Comment by unemployedsd** [comment]
Yes! Burn

**Comment by Mdlandla** [comment]
Yes  Burn

**Comment by arekadiuszkali** [comment]
Yes Burn

śr., 20 lis 2024 o 17:13 Mdlandla ***@***.***> napisał(a):

**Comment by CROkingJoel994** [comment]
yes burn

**Comment by CROkingJoel994** [comment]
create a partnership with xrp ripple, next open up a few atms for cronos investors to deposit and withdrawal with crypto.com specific location for atm will be located outside of the crypto.com arena and implementing a 5 percent burning mechanism per utilizations of atm.

**Comment by CROkingJoel994** [comment]
Approach East Los Angeles Community College and create internships opportunities for undergraduate students to not only become more recognized but also a huge engagement within the community and college campuses all over Socal
